omod/src/main/resources/apps/overallActions_extension.json

```diff
@@ -29,5 +29,15 @@
         "icon": "icon-link",
         "order": 13,
         "requiredPrivilege": "Task: coreapps.mergeVisits"
+    },
+    {
+        "id": "${project.parent.groupId}.${project.parent.artifactId}.deletePatient",
+        "extensionPointId": "patientDashboard.overallActions",
+        "type": "script",
+        "label": "coreapps.task.deletePatient.label",
+        "script": "delPatient.showDeletePatientCreationDialog({{patient.uuid}})",
```

Comment:
FINALLY WORKED, thanks to @psbrandt. `"var uid = String(/{{patient.uuid}}/); delPatient.showDeletePatientCreationDialog(uid.substring(1, uid.length-1))",`